docs/examples/model_masonry.rst

```diff
@@ -1,3 +1,46 @@
 ********************************************************************************
 Model Masonry
 ********************************************************************************
+
+.. figure:: /_images/model_masonry.png
+    :figclass: figure
+    :class: figure-img img-fluid
+
+
+Model and element creation from an obj file:
+
+>>> from compas.files import OBJ
+>>> from compas_model.viewer import ViewerModel
+>>> from compas_model.elements import Block
+>>> from compas_model.model import Model
+>>> from compas_model.algorithms import Collider
+>>> from compas.datastructures import Mesh
+>>> 
+>>> obj = OBJ('data/cross_vault.obj')
+>>> obj.read()
+>>> 
+>>> model = Model()
+>>> elements = []
+>>> for name in obj.objects:
+>>>     mesh = Mesh.from_vertices_and_faces(*obj.objects[name])
+>>>     block = Block(closed_mesh=mesh)
+>>>     elements.append(block)
+>>>     model.add_element("my_block", block)
```

Comment:
an actual interactive session doesn't look like this. just use a regular code block